diagnostic.md

```diff
@@ -54,7 +66,21 @@ How would we reference each of the following categories of elements?
 -   All `p` elements whose parent elements are `div`s
 
 <!-- your answer starts here -->
+  big {
+
+  }
 
+  #contentPane {
+
+  }
+
+  .important .red {
```

Comment:
`.important.red` is the correct way. When you add the space between `.class` names, it applies the styling to both of those class names. When you have them back to back with no space, it applies a style to elements that contain BOTH of those classes.